migrations/versions/4499b94e5e6_added_speakers_table.py

```diff
@@ -0,0 +1,38 @@
+"""Added speakers table
```

Comment:
Small nit: migration descriptions should be like commit subjects; they should describe what they do as they run. The tense should be changed here to "add" rather than "added." Since this migration will need to be updated to include a data migration as well, mentioning each aspect of the migration in the description can make it overly chatting. Focusing on the bigger picture helps.

> Add co-presenters.

The period was included to be consistent with other migrations (some are missing it, but most have it).